feature-libs/product-configurator/rulebased/cpq/cpq-configurator-normalizer.spec.ts

```diff
@@ -460,6 +462,24 @@ describe('CpqConfiguratorNormalizer', () => {
     expect(values.length).toBe(0);
   });
 
+  it('should use attribute name when attribute label is not available', () => {
+    const attributeList: Configurator.Attribute[] = [];
+    const cpqAttributeWithoutLabel: Cpq.Attribute = {
+      ...cpqAttribute,
+      label: undefined,
+      name: 'AttributeName',
+    };
+    cpqConfiguratorNormalizer.convertAttribute(
+      cpqAttributeWithoutLabel,
+      cpqGroupId,
+      CURRENCY,
+      attributeList
+    );
+    const attribute: Configurator.Attribute = attributeList[0];
+    expect(attributeList.length).toBe(1);
+    expect(attribute.label).toBe('AttributeName');
+  });
```

Comment:
this does not seem to be related to 'move value from input field'. did you fix multiple issues with this PR?

Reply:
Yes. Several issues (directly or indirectly related to handling of input fields) are fixed with this pull request. All adjustments are mentioned in the pull request comment.